### DISCUSSION TITLE
[feature] add authenticator like app '2fauth' to vaultwarden

### DISCUSSION THREAD

**Discussion by luckyluc74** [dani-garcia/vaultwarden#4743] [Ideas]
Would be nice feature to add 'web'authenticator to vaultwarden like app '2fauth'

[demo of 2fauth](https://demo.2fauth.app/login)
[github of 2fauth](https://github.com/Bubka/2FAuth)

**Comment by BlackDex** [comment]
This is already built-in into the web-vault and all other apps.
Not sure why this should be added to Vaultwarden as a separate item actually.

**Reply by luckyluc74** [reply]
Indeed, it is in Authenticator key (TOPT) , never knew this and this works great :)
Ok disregard my feature request because it is already in vaultwarden.
